README.md

```diff
@@ -85,3 +85,16 @@ Settings:
 * Weight: 100 (min)
 * Grade: 0 (middle)
 * Optical Size: 48 (max)
+
+### Testing
+
+We test the contents of our docs via [`testbook`](https://github.com/nteract/testbook).
+These tests stand up real infrastructure by executing cells in the documentation
+and expect a GCP credential to live at `~/.gcp/doc-test-creds.json`. We'll migrate
```

Comment:
I don't think requiring a specific path is needed. Lets use default credentials and if you want to provide GCP credentials then use env variable GOOGLE_APPLICATION_CREDENTIALS to provide path (gcloud will respect that env variable when set). 

https://cloud.google.com/docs/authentication/application-default-credentials#GAC

Reply:
💡 I should execute locally using SA impersonation. That'll give the same experience of the SA running it

Reply:
I see, I thought that if you provide GOOGLE_APPLICATION_CREDENTIALS pointing to path of SA key that it would be the same as impersonating that SA. Is that not true? Also when you use Workload Identity Pools there is no need to impersonate a SA, I think it will just pickup the default credentials afair.

Reply:
iirc `GOOGLE_CREDENTIALS` is the one where you provide teh entirety of the key contents. I'm saying, I was also using the downloaded SA key as an opportunity to get the roleset correct for an SA that we would use for this task but I shouldn't have bothered - impersonation allows you to do the same thing AND would have this code using a WI compatible auth path.